libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -824,6 +824,9 @@ public void onMediaUploadSucceeded(final String localMediaId, final MediaFile me
                 AttributesWithClass attributesWithClass = new AttributesWithClass(
                         content.getElementAttributes(predicate));
                 attributesWithClass.removeClass(ATTR_STATUS_UPLOADING);
+                if (mediaFile.isVideo()) {
```

Comment:
oh totally: it is. Just realized it was missing there only - should I create another PR for this change, if that makes sense?